ICAFIX/scripts/Compile_MATLAB_code.sh

```diff
@@ -72,7 +59,7 @@ compile_prepareICAs()
 
 	log_Msg "Compiling ${app_name} application"
 	${MATLAB_HOME}/bin/mcc -m -v ${app_name}.m \
-				  -I ${HCPPIPEDIR}/global/matlab/gifti-1.6 \
+				  -I ${HCPCIFTIRWDIR} \
 				  -I ${HCPPIPEDIR}/global/matlab \
```

Comment:
Now that we've eliminated the `@gifti` symlink, we could make the path order for the compilation consistent with the order that we are using in setting the path (via `addpath`) for the interpreted matlab.  That order would be:
```
...
-I ${HCPPIPEDIR}/global/matlab \
-I ${HCPCIFTIRWDIR} \
...
```
i.e., give `${HCPPIPEDIR}/global/matlab` precedence (given that -I appends).